moda-contracts/src/interfaces/ICatalog.sol

```diff
@@ -0,0 +1,336 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.21;
+
+/**
+ * @title IRegistry
+ * @dev Interface for Registry
+ */
+interface ICatalog {
```

Comment:
I don't think we need chainId in the getCatalogInfo call. The caller should know what chain they are on or what chain they are calling

Reply:
`ICatalog` is not really doing anything related to a "Catalog". `IVersionInfo` is more in line with its functionality. It makes it more reusable too.

The fields can be changed to `name` and `version`

Reply:
Maybe `getCatalogInfo` -> `versionInfo` ?

Reply:
ok updated.